packages/monocdk-experiment/gen.js

```diff
@@ -34,7 +34,7 @@ const exclude_files = [
 async function main() {
   const outdir = path.join(await fs.mkdtemp(path.join(os.tmpdir(), 'monocdk-')), 'package');
 
-  const srcdir = path.join(outdir, 'src');
+  const srcdir = outdir;
```

Comment:
Why not just use `outdoor` instead of aliasing?

Reply:
reverted